pkg/util/BUILD.bazel

```diff
@@ -3,7 +3,9 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 go_library(
     name = "go_default_library",
     srcs = [
+        "amd64.go",
```

Comment:
nit: The filenames amd64.go and s390x.go could be more descriptive. I suggest renaming them to se_amd64.go and se_s390x.go to better reflect their purpose